### PR TITLE
Add error message to replace PORTABLE_EXE with EXECUTABLE

### LINKED ISSUE #4757: Make better error message/suggestion for PORTABLE_EXE
PORTABLE_EXE is removed, so job scripts with this keyword will now fail, should give an indication to use EXECUTABLE instead

### PR DESCRIPTION
**Pull request by DanSava**
**Issue**
Resolves #4757

## Pre review checklist

- [x] Added appropriate release note label
- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [x] Updated documentation
- [x] Ensured new behaviour is tested

Adding labels helps the maintainers when writing release notes. This is the [list of release note labels](https://github.com/equinor/ert/labels?q=release-notes).

### CODE REVIEW

**Comment by oysteoh** [comment]
Very good! This should also be bacported into the 2023.02 release i think as this is a recurring question from users!

File: src/ert/_c_wrappers/job_queue/ext_job.py [hunk 1]
```diff
@@ -178,6 +178,22 @@ def from_config_file(cls, config_file: str, name: Optional[str] = None):
...
+                (i_err for i_err, err in enumerate(conf_err.errors) if err_msg in err),
+                None,
+            )
+            if err_index is None:
+                raise conf_err from None
+            with open(config_file, encoding="utf-8") as f:
+                if "PORTABLE_EXE " in f.read():
+                    conf_err.errors[err_index] = conf_err.errors[err_index].replace(
+                        err_msg,
+                        '"PORTABLE_EXE" key was deprecated, '
```
Comment (oysteoh) [inline src/ert/_c_wrappers/job_queue/ext_job.py:193]: I think "PORTABLE_EXE" key is deprecated, ...  is better than "was" .... ?

**Review by oysteoh — APPROVED** [review]
One small comment regarding wording of the message, other than that - very nice :+1:

**Comment by oysteoh** [comment]
Errrors are related to changes merged yesterday i think - https://github.com/equinor/ert/pull/4743

**Comment by DanSava** [comment]
Jenkins test this please

**Comment by oysteoh** [comment]
> Jenkins test this please

I guess you need a new build of bleeding in order to get it to pass on jenkins?

**Pull request merged**